.github/workflows/preview-agent.yaml

```diff
@@ -37,8 +39,9 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Cleanup preview agent
-        run: npm run cleanup-agent -- --pr ${{ github.event.number }}
+        run: npm run cleanup-agent -- --pr ${{ github.event.number }} --repository ${{ github.repository }} --branch ${{ github.head_ref }}
```

Comment:
- remove repository argument since we know which repo we're always in
- remove the branch argument since we could look it up with the pr event number